WordPress/Classes/Models/AbstractPost.m

```diff
@@ -581,6 +581,17 @@ - (BOOL)hasRemoteChanges
             || self.remoteStatus == AbstractPostRemoteStatusFailed);
 }
 
+- (BOOL)hasVersionConflict
+{
+    AbstractPost *original = (AbstractPost *)self.original;
+    if (original.dateModified) {
+        return ![self.dateModified isEqualToDate:original.dateModified] && self.hasLocalChanges;
+    } else {
+        // This is to accommodate tests - many of the mock Posts do not have an original with a dateModified property
```

Comment:
This is not only for tests. :)

When you create a new post in the app, it will not have the `original`.